### PR TITLE
libjxl, revbump, enable 32bit/documentation

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: media-libs/libjxl/libjxl0.11-0.11.0.recipe [hunk 1]
```diff
@@ -12,22 +12,34 @@ LICENSE="BSD (3-clause)
...
 	Zlib"
-REVISION="1"
+REVISION="2"
 SOURCE_URI="https://gitlab.com/wg1/jpeg-xl/-/archive/v$portVersion/jpeg-xl-v$portVersion.tar.bz2"
 CHECKSUM_SHA256="378ebfee94e7d200db8c8dbbdf7b2c9b99367348c4c8f16dfb8a309a0a259c4c"
+SOURCE_DIR="jpeg-xl-v$portVersion"
+PATCHES="libjxl0.11-$portVersion.patchset"
+
 srcGitRev_2="42030a771244ba67f86b1c1c76a6493f873c5f91"
 SOURCE_URI_2="https://codeberg.org/Begasus/ckcms/raw/branch/master/ckcms-20231106.tar.gz"
```
Comment (korli) [inline media-libs/libjxl/libjxl0.11-0.11.0.recipe:22]: isn't it skcms?
```suggestion
SOURCE_URI_2="https://codeberg.org/Begasus/ckcms/raw/branch/master/ckcms-20231106.tar.gz"
```

Comment (Begasus) [reply media-libs/libjxl/libjxl0.11-0.11.0.recipe:22]: Oh shoot! Need to rename the clone upstream then, good catch!

Comment (Begasus) [reply media-libs/libjxl/libjxl0.11-0.11.0.recipe]: @korli updated repo name and archive, what is the change you suggested, can't spot the difference :)

File: media-libs/libjxl/libjxl0.11-0.11.0.recipe [hunk 2]
```diff
@@ -76,43 +75,67 @@ CONFLICTS_devel="
...
+	lib:libavif$secondaryArchSuffix
+	lib:libgif$secondaryArchSuffix
+	lib:libIex_3_2$secondaryArchSuffix
+	lib:libIlmThread_3_2$secondaryArchSuffix
+	lib:libImath_3_1$secondaryArchSuffix
+	lib:libOpenEXR_3_2$secondaryArchSuffix
+	lib:libOpenEXRUtil_3_2$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+#	lib:libstdc++$secondaryArchSuffix
```
Comment (Begasus) [inline media-libs/libjxl/libjxl0.11-0.11.0.recipe:96]: Together with libatomic below it looks like these are not used/required, maybe they did before?

**Comment by Begasus** [comment]
Still no luck on opening a *.jxl file though :(

**Pull request merged**